.goreleaser.yaml

```diff
@@ -58,10 +58,11 @@ archives:
     # use zip for windows archives
     format_overrides:
       - goos: windows
-        format: zip
+        formats: [ 'zip' ]
 
 kos:  # See https://goreleaser.com/customization/ko/
-  - repository: ghcr.io/refaktor/rye
+  - repositories:
+    - ghcr.io/refaktor/rye
```

Comment:
If you want to push images also to https://hub.docker.com/r/refaktorlabs/ryelang you could add it here, but credentials will need to be set some way.

Reply:
See https://goreleaser.com/customization/dockerhub/

Reply:
But that can be done separately

Reply:
No need to rush it into this PR.